_includes/footer.html

```diff
@@ -12,7 +12,9 @@
 				</div>
 			</div>
 			<div class="copyright text-muted col-6 mr-0 my-auto">
-				<div class="row" style="float: right">Dev Club</div>
+				<div class="row" style="float: right">
+					<img src="DevClubLogo.svg" height="90%">
```

Comment:
Change this from `DevClubLogo.svg` to `{{ site.url }}/DevClubLogo.svg` and yours will be good to merge